### ISSUE TITLE
SVG files are not rendered properly

### ISSUE THREAD

**Issue by petaflot** [OpenShot/openshot-qt#4423]
**Describe the bug**
When adding slides such as titles and editing them with inkscape (the default?), I noticed centered text is _not_ centered.

I didn't check further for other bugs.

**Steps to reproduce the behavior:**
1a. edit a default title "slide", set the text and use the "centered text" option
1b. make an SVG from scratch with a centered text, to use as a frame (or group of frames)
2. import it
3. watch the result : the text is way off the left side of the frame, not centered

**Expected behavior**
the SVG should be displayed in openshot the way it is displayed in ie. inskcape

**Workaround**

Open each SVG file individually, export it as PNG with inkscape (with the desired resolution) and import this instead. While this would be easy to script within openshot, it's quite a pain to do manually - especially when an SVG is modified eg. when a typo is found.

**System Details**
- OpenShot Version: 2.6.1-dev
- libopenshot Version:  0.2.7-dev
- Platform: Linux-5.10.49-gentoo-r1-x86_64-x86_64-Intel-R-_Core-TM-_i5-7600_CPU_@_3.50GHz-with-gentoo-2.7
- Processor: Intel(R) Core(TM) i5-7600 CPU @ 3.50GHz
- Machine: x86_64
- Python version: 3.6.9
- Qt5 version: 5.9.5
- PyQt5 version: 5.10.1
- Qt Detected Languages: 
- LANG Environment Variable: 
- LOCALE Environment Variable: 
- Daily Build: Verified issue still exists in daily build: http://github.com/OpenShot/openshot-qt/releases/download/OpenShot-v2.6.1-dev-daily-8289-21597c8a-73c519f0-x86_64.AppImage

**Log Files**
- [openshot-qt.log](https://s3.amazonaws.com/logs.openshot.org/5h8pq4myedgpd33am6dias6jx2d021ex/openshot-qt.zip) (3 KB)
- libopenshot.log: File too small or empty (not uploaded)

**Exception / Stacktrace**
```No stacktrace found in log files```

**Comment by ferdnyc** [comment]
Hmm. This may be a font-replacement issue. IIRC when Inkscape centers text, if not told otherwise (there are multiple ways) it inserts a text bounding box and centers **that**. So, if the text is rendered with a font that has a different width, it can end up off center.

One question is exactly **HOW** you're centering the text, since like I said there are different ways to do that in Inkscape. A sample SVG file would be helpful, but I'll try to play with this and see if I can reproduce.

**Comment by petaflot** [comment]
Here's one:

http://dpaste.com/G25U43QGD

Since I installed a new font yesterday, I restarted openshot ; by misake I launched the packaged 0.2.5-r1 and noticed the alignment was mostly worse (text was also vertically offset) EXCEPT for that new font* I had just installed (for the rest I use NoTo Sans).

http://dpaste.com/B8VQBHL8X

With the daily nightbuild from two days ago, that new font is also completely offset (in fact so offset it's out of the canvas)

* https://www.1001fonts.com/tommys-first-alphabet-font.html

**Comment by petaflot** [comment]
I tried to render once.. just to see how it would turn out.

I noticed on this one title slide, the text is rendered right-aligned instead of centered...

http://dpaste.com/7PF73AJY5

As for how I center the text, I create a text block, type my text, select all of the text and chose the "center text" entry in the toolbar's dropdown. Then I hit F1, select the text block, and center it relative to the page.
